README.md

```diff
@@ -377,7 +377,7 @@ object ApplicationConfig extends GenericReader
 
 You add this library as a sbt dependency:
 ```scala
-libraryDependencies += "org.zalando" %% "grafter" % "1.3.1"
+libraryDependencies += "org.zalando" %% "grafter" % "1.4.2"
```

Comment:
Yes this is quite annoying. This is where you would like a bit of macros in your markdown.